roles/create_certification_project/templates/create_project_helmchart.json.j2

```diff
@@ -1,21 +1,28 @@
 {
   "certification_status": "Started",
-  {% if cert_settings is defined and cert_settings.email_address is defined %}
   "contacts": [
     {
       "email_address": "{{ cert_settings.email_address }}",
       "type": "Technical contact"
     }
   ],
-  {% endif %}
   "helm_chart": {
+    "application_categories": [
+    "{{ cert_settings.application_categories }}"
+    ],
     "chart_name": "{{ cert_item.chart_name }}",
-    "distribution_method": "undistributed",
+    "distribution_instructions": "{{ cert_settings.distribution_instructions }}",
+    "distribution_method": "{{ cert_settings.distribution_method }}",
+    "github_usernames": [
+    "{{ cert_settings.github_usernames }}"
+    ],
+    "long_description": "{{ cert_settings.long_description }}",
+    "short_description": "{{ cert_item.short_description }}"
   },
   "marketplace": {
     "enablement_status": "ineligible"
   },
   "name": "{{ cert_item.chart_name }}",
   "project_status": "active",
   "type": "Helm Chart"
-}
+}
```

Comment:
empty line at the end of file is required

Reply:
extra empty line is added to the end of the json file.